packages/react/tsconfig.json

```diff
@@ -2,11 +2,13 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "isolatedModules": false,
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "types": ["jest"]
```

Comment:
what does it mean?

Reply:
To declare the jest functions for ts and avoid errors in the editor like this -
![image](https://github.com/frontegg/frontegg-react/assets/106734943/126647ed-ad00-4f51-9c2e-9dbd6bc2854a)